core/domain/exp_jobs_one_off_test.py

```diff
@@ -290,6 +292,132 @@ def test_first_published_time_of_exploration_that_is_unpublished(self):
             exp_first_published, exploration_rights.first_published_msec)
 
 
+class ExpSummariesContributorsOneOffJobTest(test_utils.GenericTestBase):
+
+    ONE_OFF_JOB_MANAGERS_FOR_TESTS = [
+        exp_jobs_one_off.ExpSummariesContributorsOneOffJob]
+
+    EXP_ID = 'exp_id'
+
+    USERNAME_A = 'usernamea'
+    USERNAME_B = 'usernameb'
+    EMAIL_A = 'emaila@example.com'
+    EMAIL_B = 'emailb@example.com'
+
+    def setUp(self):
+        super(ExpSummariesContributorsOneOffJobTest, self).setUp()
+
+    def test_contributors_for_valid_contribution(self):
+        """Test that if only one commit is made, that the contributor
+        list consists of that contributor's user id.
+        """
+        self.signup(self.EMAIL_A, self.USERNAME_A)
+        self.user_a_id = self.get_user_id_from_email(self.EMAIL_A)
+
+        exploration = self.save_new_valid_exploration(
+            self.EXP_ID, self.user_a_id)
+        job_id = exp_jobs_one_off.ExpSummariesContributorsOneOffJob.create_new()
+        exp_jobs_one_off.ExpSummariesContributorsOneOffJob.enqueue(job_id)
+        self.process_and_flush_pending_tasks()
+
+        exploration_summary = exp_services.compute_summary_of_exploration(exploration)
+        self.assertEqual(
+            [self.user_a_id], exploration_summary.contributor_ids)
+
+    def test_repeat_contributors(self):
+        """Test that if the same user makes more than one commit that changes
+        the content of an exploration, the user is only represented once in the
+        list of contributors for that exploration.
+        """
+        self.signup(self.EMAIL_A, self.USERNAME_A)
+        self.user_a_id = self.get_user_id_from_email(self.EMAIL_A)
+
+        # have one user make two commits
```

Comment:
Two small nits on comments:
- all comments should start with a capital letter and end with a period
- suggest using "ids", not "id's" -- plurals don't have apostrophes